tutorial/source/tensor_shapes.ipynb

```diff
@@ -273,12 +274,12 @@
     "    x_axis = pyro.iarange(\"x_axis\", 3, dim=-2)\n",
     "    y_axis = pyro.iarange(\"y_axis\", 2, dim=-3)\n",
     "    with x_axis:\n",
-    "        x = pyro.sample(\"x\", Normal(0, 1).expand_by([3, 1]))\n",
+    "        x = pyro.sample(\"x\", Normal(0, 1))\n",
```

Comment:
Are we relying on the automatic broadcasting in `TraceEnum_ELBO` here?

Reply:
Oops, yes, but I'll add an explicit `@poutine.broadcast`

Reply:
We can also link to the section below for the out of order appearance of `poutine.broadcast`.